executor/explainfor_test.go

```diff
@@ -1574,3 +1574,25 @@ func TestIssue28792(t *testing.T) {
 	r2 := tk.MustQuery("EXPLAIN SELECT t12.a, t12.b FROM t12 LEFT JOIN t97 use index () on t12.b = t97.b;").Rows()
 	require.Equal(t, r2, r1)
 }
+
+func (s *testSerialSuite) TestExplainForConnectionBrief(c *C) {
```

Comment:
We are migrating test-infra to testify. Could you do that for the test case? You can take a look at other test cases in the file as examples.

Reply:
> We are migrating test-infra to testify. Could you do that for the test case? You can take a look at other test cases in the file as examples.

PTAL